bootstrap.php

```diff
@@ -19,6 +22,66 @@
 	}
 	return;
 }
+
+
+/*
+ * Initialize coming soon module via container
+ */
+$nfd_module_container = new Container();
+// Set plugin to container
+$nfd_module_container->set(
+	'plugin',
+	$nfd_module_container->service(
+		function() {
+			return new Plugin(
+				array(
+					'id'   => 'hostgator',
+					'file' => HOSTGATOR_PLUGIN_FILE,
+				)
+			);
+		}
+	)
+);
+
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
```

Comment:
Can we just add an upgrade routine to use `nfd_coming_soon` instead?

Reply:
Done